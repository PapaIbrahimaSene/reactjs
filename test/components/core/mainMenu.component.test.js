import React from 'react';
import { shallow } from 'enzyme';
import { MainMenu } from '../../../src/components/core/mainMenu.component';

test('should create the Main Menu bar', () => {
    const wrapper = shallow(<MainMenu />);

    expect(wrapper).toMatchSnapshot();
});