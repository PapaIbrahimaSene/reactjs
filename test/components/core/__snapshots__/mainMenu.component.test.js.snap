// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create the Main Menu bar 1`] = `
<Menu
  color="orange"
  fixed="top"
  inverted={true}
>
  <Container>
    <MenuItem
      as="a"
      header={true}
    >
      <Icon
        as="i"
        inverted={true}
        name="hacker news"
        size="big"
      />
      HackerNews API ReactJS Sample
    </MenuItem>
  </Container>
</Menu>
`;
